### PR TITLE
fix template not found

### PR DESCRIPTION
**Pull request by guileen**
if template not found return status 500 template not found.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -160,6 +160,12 @@ Hbs.prototype.createRenderer = function() {
     var tplPath = hbs.getTemplatePath(tpl),
       template, rawTemplate, layoutTemplate;
 
+    if(tplPath === undefined) {
+      this.status = 500;
```
Comment (shellscape) [inline index.js]: this isn't a pattern I'm familiar with koa-hbs using. personally think it'd be better to `throw` here than anything. `status = 500` doesn't really mean anything, does it? only thing that comes to mind is an http 500 error, which wouldn't be 'not found'.

Comment (jwilm) [inline index.js]: @shellscape thank you for starting a discussion about this. I looked into this a bit more, and I agree with your suggestion. #42 is the tracking issue to utilize the standard koa error handling mechanisms.

Comment (jwilm) [inline index.js]: I meant to mention, there won't be a release with this change. There will be a new minor version once #42 closes.

**Comment by shellscape** [comment]
Will be implementing a proper `throw` with version 0.9.0.

**Pull request closed without merging**